### ISSUE TITLE
Advice: Binary Classification with PointNet

### ISSUE THREAD

**Issue by shrutirao94** [charlesq34/pointnet#252]
Thank you for sharing this pointnet repository! It has been really useful to me!

As part of my masters research, I wanted to use it for binary classification of point clouds for neutrino data (neutrino point clouds versus noise point clouds). I was wondering if you had any comments on the required modifications to the existing architecture? 

I have a basic understanding of deep learning and think I should be changing the activation function to _sigmoid_ and using a _binary cross entropy loss function_ as well. Are there any other factors I should be aware of? Any advice would be great! I have two classes as I mentioned previously with a large number of samples. I was able to run the code but got inaccurate results since I did not modify the architecture for binary classification!

Thank you!
